web/css/src/themes/default.css

```diff
@@ -1330,6 +1331,13 @@
 	}
 }
 
+/* Web Terminal
+   ========================================================================== */
+.js-web-terminal {
```

Comment:
Ideally `.js-` classes would only be used by the JS. I would go with `.web-terminal` here.

Reply:
so should I add both `.js-web-terminal` or just `.web-terminal`? Honestly not sure about the padding's look anyways.

Reply:
Alright! Changed it:)